src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  @extend %flexAlignCenter;
+  flex-direction: column;
+
+  width: $width-card;
+  padding: 32px 16px 16px;
+
+  background: $color-bcg-white;
```

Comment:
but we can use this color for `color\border-color\etc: $color-bcg-white`, so let's remove relation in name with background))

just set the name the same way we name the color

F.e.

```SCSS
$c-red
$c-light-blue
$c-dark-gray
```

or we can use color saturation in the name
<img width="144" alt="image" src="https://user-images.githubusercontent.com/86183007/216517467-c9b38dd8-5ad4-4ea3-8960-e58b36391ec7.png">